templates/repo/view_list.tmpl

```diff
@@ -26,6 +26,23 @@
 						{{end}}
 				</a>
 				<span class="grey has-emoji">{{RenderCommitMessage false .LatestCommit.Summary .RepoLink $.Repository.ComposeMetas}}</span>
+				{{if .LatestCommitStatus}}
```

Comment:
Can this code be moved to separate template and reused in both places?

Reply:
done.